src/SelectedMovie.js

```diff
@@ -1,36 +1,52 @@
-import React from 'react';
+import React, { Component } from 'react';
 import './SelectedMovie.css';
 
-const SelectedMovie = () => {
-  let num = 63000000;
-  let mins = getTime(139);
-  let date = getDate('2019-12-04');
-  return (
-    <main className='selected-page'>
-      <img className='selected-poster' src = 'https://image.tmdb.org/t/p/original//7G2VvG1lU8q758uOqU6z2Ds0qpA.jpg' />
-      <section className='selected-details'>
-       <h2>Fake Movie Title</h2>
-       <p className='movie-detail overview'>Some overview that is full of buzzwords to attempt to entice you to watch this movie! Explosions! Drama! True love! Robots! A cute dog!</p>
-       <p className='movie-detail'><strong>Tagline: </strong>It's a movie!</p>
-       <p className='movie-detail'><strong>Rating: </strong>⭐️ {6}/10</p>
-       <p className='movie-detail'><strong>Release Date: </strong>{date}</p>
-       <p className='movie-detail'><strong>Budget: </strong>${num.toLocaleString("en-US")}</p>
-       <p className='movie-detail'><strong>Revenue: </strong>${num.toLocaleString("en-US")}</p>
-       <p className='movie-detail'><strong>Runtime: </strong>{mins}</p>
-      </section>
-    </main>
-  )
-}
+class SelectedMovie extends Component {
+  constructor() {
+    super()
+    this.state = {
+      movieInfo: {}
+    }
+  }
 
-const getTime = (mins) => {
-  const hours = Math.floor(mins / 60);
-  const remainingMins = mins % 60;
-  return `${hours} ${(hours > 1) ? 'hours' : 'hour'} ${remainingMins} minutes`
-}
+  componentDidMount() {
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies/'+ this.props.id)
+    .then(res => res.json())
+    .then(data => this.setState({movieInfo: data.movie}))
+  }
+
+  getDate = (dateStr) => {
+    let date = new Date(dateStr);
+    // const month = new Intl.DateTimeFormat('en-US').format(date);
+    return `${date.getDay()}, ${date.getFullYear()}`
+  }
+  
 
-const getDate = (dateStr) => {
-  const date = new Date(dateStr);
-  return `${new Intl.DateTimeFormat('en-US', {month: 'long'}).format(date.getMonth())} ${date.getDay()}, ${date.getFullYear()}`
+  getTime = (mins) => {
+    const hours = Math.floor(mins / 60);
+    const remainingMins = mins % 60;
+    return `${hours} ${(hours > 1) ? 'hours' : 'hour'} ${remainingMins} minutes`
+  }
+  
+  render() {
+    const movie = this.state.movieInfo;
+    return (
+      <main className='selected-page'>
+        <img className='selected-poster' src={movie.poster_path} />
+        <section className='selected-details'>
+         <h2>{movie.title}</h2>
+         <p className='movie-detail overview'>{movie.overview}</p>
```

Comment:
Generally, the `<strong>` tag is a semantic tag used for emphasis; it means that the text is important, urgent, or serious and "should not be used to apply bold styling; use the CSS font-weight property for that purpose." (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong)

I'll let y'all determine if its use in this section counts as semantic or not - because it might. I just wanted to put this in as food for thought.